Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
+    m "But sometimes, I watch a movie."
+    m "You know, just to pass the time ahaha."
+    menu:
+        "Wow, really?":
+            m "Is it really that surprising [player]?"
+            menu:
+                "Kinda.":
+                    m "Well, I guess you're not the first one to think that."
+                    m "People often think I'm for not interested in movies some reason."
+                    m "Maybe because I was in a literatire club?"
+                    m "And a debate club before that? I don't know."
+                    m "But that's beside the point. Anyway..."
+                "Not really.":
+                    m "Well I {i}am{/i} just a teenager after all."
+                    m "I can watch a movie too ahaha."
+                    m "So anyways..."
+                "I thought you were more interested in reading than watching.":
+                    m "Well, most of the time I am."
+                    m "But sometimes, even I'd get tired of only reading."
+                    m "Sometimes I just feel like watching a random movie."
+                    m "But I don't blame you for being surprised [player]."
+        "That's pretty normal":
+            m "Well, even I get bored waiting ahaha."
+            m "I-It's not that I'm demanding you to spend more time with me...I understand that you're busy in your reality too."
+            m "But I'm not saying you shouldn't {i}not{/i} spend time with me...{w} It's just..."
+            m "Uhh...{w} How do I explain this?"
+            menu:
+                "You can do it!~":
+                    m "Don't be such a tease [player]~"
+                    m "Ahaha~"
+                "You don't have to force yourself.":
+                    m "I'm not really forcing myself [player]."
+                    m "I just need a bit of time to gather my thoughts."
+                    m "..."
+                "Okay, everyone! It's time to explain yourself~":
+                    m "Pfft!"
+                    m "Ahahahaha!"
+                    m "That's a nice impression of me [player]!"
+                    m "So, where was I?{w} Oh yeah! {i}explaining myself{/i}~"
+                    m "Ahaha~"
+            m "You know that I love it when you come to spend time with me right? And I'm always happy waiting for you to come here."
+            m "But it get's boring sometimes and I pass the time. You get that, right [player]?
+            m "This is kinda awkward.{w}Umm...{w}Movies! Where was I?"
+        "You mean like adaptations of your favourite books?":
+            m "Well, usually yes."
+            m "But I also watch regular movies from time to time."
+            m "There are some pretty interesting movies out there too."
+            m "For example."
+    m "While you were gone, I watched this movie called '{i}Whiplash{/i}'."
+    m "It was pretty good."
+    m "Maybe you know the movie too [player]?"
+    menu:
+        "Yes."
```

Comment:
missing `:` at the end of `        "Yes"`

Reply:
Done